software/control/core/core.py

```diff
@@ -1610,1207 +1311,23 @@ def add_current_coords_to_focus_map(self):
         self.focus_map_coords.append((x,y,z))
         print(f"Added triple ({x},{y},{z}) to focus map")
 
-
-class MultiPointWorker(QObject):
```

Comment:
It might make sense to move this back from core.multipoint.py just so we can see what actually changed, and then break core.py apart into pieces in multiple subsequent pr.

Reply:
Perhaps it’s time to make this independent of pyqt (remove dependence on signaling, for example for updating the illumination and channel settings)?